src/main/scala/com/databricks/spark/xml/XmlReader.scala

```diff
@@ -19,10 +19,11 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.{DataFrame, SQLContext}
 import org.apache.spark.sql.types.StructType
 import com.databricks.spark.xml.util.XmlFile
+import org.apache.spark.sql.Row;
 
 /**
- * A collection of static functions for working with XML files in Spark SQL
- */
+  * A collection of static functions for working with XML files in Spark SQL
+  */
```

Comment:
Spark itself follows Java style comment. Should be better to keep it consistent with Apache Spark.